src/apps/SocketsRegistry/DetailsView/SocketDetailInfoBar.js

```diff
@@ -48,7 +46,7 @@ class SocketsRegistryInfoBar extends Component {
       sidebarContent: {
         left: 0,
         position: 'absolute',
-        top: 300
+        top: 188
```

Comment:
Yup, I know but @wpasinski is removing whole socket search bar in his task and we'll adjust sidebar position later.